mne/preprocessing/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+
+# License: BSD (3-clause)
+
+import numpy as np
+
+from ..utils import logger
+
+
+# adapted from SciPy
+
+def check_COLA(win, nperseg, noverlap, tol=1e-10):
```

Comment:
check_COLA -> check_cola or better

_check_overlap_add_constraint (private)

Reply:
I'd rather keep it this way to mirror SciPy more closely / make translations back and forth easier

Reply:
k